adhocracy4/phases/models.py

```diff
@@ -20,6 +20,29 @@ def active_phases(self):
     def finished_phases(self):
         return self.filter(end_date__lte=timezone.now())
 
+    def past_phases(self):
```

Comment:
good point. we somehow missed the duplication between finished and past phases. btw. finished_phases is not in any project, just in the a4 tests. i think i'd be save to remove it.

we had many discussions about the ordering. 
@xi wished for a consistent api regarding ordering (by setting Meta.ordering = 'start_date')
while i tried to keep backwards compatibility. Changing the default ordering would require to adapt the dashboard code to explicit order on the software defined weight.
i think it would be a good idea but wanted to hear back form you what you think about breaking